.github/workflows/docs-pdf.yml

```
@@ -48,15 +53,20 @@ jobs:
             Automated refresh from **Docs PDF** workflow ([`.github/workflows/docs-pdf.yml`](.github/workflows/docs-pdf.yml)).
 
+            Triggered by: ${{ github.event_name }} on `${{ github.ref_name }}`.
+
             Merge when ready so `pdf/open-fdd-docs.pdf` and `pdf/open-fdd-docs.txt` stay current on `master`.
           add-paths: |
             pdf/open-fdd-docs.pdf
             pdf/open-fdd-docs.txt
           committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
           author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
           delete-branch: true
+          maintain-current-branch: true
```

actionlint (1.7.12)

[error] input "maintain-current-branch" is not defined in action "peter-evans/create-pull-request@v7". available inputs are "add-paths", "assignees", "author", "base", "body", "body-path", "branch", "branch-suffix", "branch-token", "commit-message", "committer", "delete-branch", "draft", "labels", "maintainer-can-modify", "milestone", "path", "push-to-fork", "reviewers", "sign-commits", "signoff", "team-reviewers", "title", "token"

(action)